docs/cel_expressions.md

```diff
@@ -206,4 +206,19 @@ interceptor.
      <pre>decodeb64(body.message.data)</pre>
     </td>
   </tr>
+  <tr>
+    <th>
+     compareSecret
+    </th>
+    <td>
+      string.compareSecret(string, string, string) -> bool
+    </td>
+    <td>
+      Constant-time comparison of strings against secrets, this will fetch the secret using the combination of namespace/name and compare the token key to the string using a cryptographic constant-time comparison..<p>
+      The event-listener service account must have access to the secret.
+    </td>
+    <td>
+     <pre>header.canonical('X-Secret-Token').compareSecret('namespace', 'secret-name', 'key')</pre>
```

Comment:
can the namespace be optional? In other interceptors, if one is not specified, it defaults to the EL's namespace.

Reply:
I implemented it with variable args, if you provide 3, the first is the namespace, only 2 and it defaults to the eventlistener's namespace.

Reply:
Ahh ok, that sounds good.